README.md

```diff
@@ -1,50 +1,70 @@
-# Prism
+[![Prism is a set of packages for API mocking with OpenAPI v2 (formerly known as Swagger) and OpenAPI v3.](./examples/readme-header.svg)](https://stoplight.io/api-mocking?utm_source=github&utm_medium=prism&utm_campaign=readme)
 
 [![CircleCI](https://img.shields.io/circleci/build/github/stoplightio/prism/master)](https://circleci.com/gh/stoplightio/prism)
 [![NPM Downloads](https://img.shields.io/npm/dw/@stoplight/prism-http?color=blue)](https://www.npmjs.com/package/@stoplight/prism-cli)
 [![Buy us a tree](https://img.shields.io/badge/Buy%20us%20a%20tree-%F0%9F%8C%B3-lightgreen)](https://offset.earth/stoplightinc)
 
-Prism is a set of packages for API mocking with **OpenAPI v2** (formerly known as Swagger) and **OpenAPI v3**.
-
-For example, given an API description document you can spin up a mock HTTP server and respond realistically based on your requests
+- **Mock Servers**: Life-like mock servers from any API Specification Document.
+- **Validation Proxy**: Contract Testing for API Consumers and Developers.
+- **Comprehensive API Specification Support**: OpenAPI 3.0, OpenAPI 2.0 (FKA Swagger) and Postman Collections support.
 
 ![Demo of Prism Mock Server being called with curl from the CLI](./examples/prism-cli.svg)
 
 > Note: This branch refers to Prism 3.x, which is the current version most likely you will use. If you're looking for the 2.x version, look at the [`2.x` branch][2.x]
 
-## Installation
+# Overview
+
+- [Installation and Usage](#-installation-and-Usage)
+- [Documentation and Community](#-documentation-and-community)
+- [Roadmap](#-roadmap)
+- [FAQs](#-faqs)
+- [Contributing](#-contributing) 
 
-Prism requires NodeJS >= 12 to properly work.
+## 🧰 Installation and Usage
 
+**Installation**
 ```bash
 npm install -g @stoplight/prism-cli
 
 # OR
 
 yarn global add @stoplight/prism-cli
 ```
+> Prism requires NodeJS >= 12 to properly work.
+
+For more installation options, see our [installation documentation](https://meta.stoplight.io/docs/prism/docs/getting-started/01-installation.md).
```

Comment:
This forces me to go on the web when I have all the documentation right in a directory. I'd say to preserve the local link.

Reply:
Fixed